charts/redis-operator/values.yaml

```
@@ -0,0 +1,41 @@
+# Default values for redis-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: redis-operator
+createNamespace: true
+
+replicaCount: 1
+
+image:
+  repository: bringg/redis-operator
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.3.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  runAsNonRoot: true
+
+securityContext:
+  runAsUser: 1001
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 192Mi
+  requests:
+    cpu: 50m
+    memory: 128Mi
```

```suggestion
resources: {}
```

Leave it to the user to decide